sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1453,7 +1453,18 @@ graph LR
     Recording -->|End Recording| Executing
 ....
 
-==== Queue Properties
+==== Implicit Queue Recording
+
+Submitting a command-group to a queue can implicitly change its state
+to `queue_state::recording`. This will occur when the command-group depends on
+an event that has been returned by a queue in the recording state.
```

Comment:
Think we should add an extra sentence clarifying that this takes effect starting on that command-group submission not after it, so the command-group is added as a node in the graph with an edge to that node the dependent event came from.